packages/instant/src/style/theme.ts

```diff
@@ -26,8 +27,10 @@ export const theme: Theme = {
     white: 'white',
     lightOrange: '#F9F2ED',
     darkOrange: '#F2994C',
+    clear: 'rgba(0, 0, 0, 0.0)',
```

Comment:
I think this just belongs with the other opacity-based values below?

Reply:
We need to figure out this color stuff... apologies if it's a bit confusing right now.